server/datastore/mysql/migrations/tables/20230920091442_PolicyStatsTable.go

```diff
@@ -0,0 +1,47 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230920091442, Down_20230920091442)
+}
+
+func Up_20230920091442(tx *sql.Tx) error {
+	stmt := `
+	CREATE TABLE policy_stats (
```

Comment:
A question around the `policy_stats` table:
Say we have a global policy with id=X and team A with ID=42 and team B with ID=99.
So we will end up with three rows for such policy:
- `policy_id=X, inherited_team_id=0`
- `policy_id=X, inherited_team_id=42`
- `policy_id=X, inherited_team_id=99`

The second and third row will contain the counts for the hosts that belong to those teams.
What about the first row? Does it contain the counts for ALL hosts or only for hosts that are not member of a team?

Reply:
Row 1 will contain counts for all hosts.  I'm not aware of any apis that count only hosts that are not a member of a team, but i'll have to rethink this schema if we do (or if we plan to in the future).